addons/mail/static/src/messaging/component/emojis_popover/emojis_popover.xml

```diff
@@ -2,9 +2,10 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.messaging.component.EmojisPopover" owl="1">
-        <div class="o_EmojisPopover">
+        <div class="o_EmojisPopover" tabindex="100">
```

Comment:
I'm not convinced either to be honest. But then we have a problem
https://w3c.github.io/uievents/#event-type-focusout

the event focusout bubbles.
the target attribute is the one loosing focus
relatedTarget is the one receiving focus (given that it is focusable) or null

so this means that for any focus transition within a container, that container will receive as many focusout events as there are transitions.

We can't always protect this by telling the handler to do nothing if both the source and the destination of the transition are inside the container since more often than not, the destination (relatedTarget) will be unknowable

This is why I put tabindex: it makes the relatedTarget known to the event focusout

Reply:
This is actually directly related to my other comment.

This conceptual issue should disappear if we don't try to stop propagation but we check just for target/relatedTarget on the focus out handler in the composer and act accordingly (ie. don't do what is bothering us if the relatedTarget is inside the emoji popover).